packages/map/src/CdcMap.js

```diff
@@ -1224,13 +1227,26 @@ const CdcMap = ({ className, config, navigationHandler: customNavigationHandler,
     }
   }, [state.general.statePicked])
 
-  // When geotype changes
+  
   useEffect(() => {
-    // UID
+    // When geotype changes - add UID
     if (state.data && state.columns.geo.name) {
       addUIDs(state, state.columns.geo.name)
     }
   }, [state])
+  
+  // DEV-769 make "Data Table" both a required field and default value
```

Comment:
Excellent job here -  Perfect use of the right property `dataTable.title`, no issues with the spread operator from what I can see, and great commenting sharing the ticket number and what the useEffect was needed for.